authorization_services/topics/policy-user-policy.adoc

```diff
@@ -1,9 +1,9 @@
 [[_policy_user]]
-= User-Based Policy
+= User-based policy
 
 You can use this type of policy to define conditions for your permissions where a set of one or more users is permitted to access an object.
 
-To create a new user-based policy, select *User* in the dropdown list in the upper right corner of the policy listing.
+To create a new user-based policy, select *User* in the item list in the upper right corner of the policy listing.
 
 .Add a User-Based Policy
 image:{project_images}/policy/create-user.png[alt="Add User-Based Policy"]
```

Comment:
Image title casing

Reply:
I think this one should remain initial capitals to match the UI.